fontbe/src/features.rs

```diff
@@ -322,13 +555,7 @@ impl<'a> VariationInfo for FeaVariationInfo<'a> {
     fn resolve_variable_metric(
         &self,
         values: &HashMap<NormalizedLocation, i16>,
-    ) -> Result<
-        (
-            i16,
-            Vec<(write_fonts::tables::variations::VariationRegion, i16)>,
-        ),
-        Box<(dyn StdError + 'static)>,
-    > {
+    ) -> Result<(i16, Vec<(VariationRegion, i16)>), Box<(dyn StdError + 'static)>> {
         let var_model = &self.static_metadata.variation_model;
```

Comment:
this is assuming that a single global VariationModel applies to all glyphs which isn't necessarily the case. We should make sparse models for each unique set of locations  (similarly to how we did for gvar and HVAR), otherwise the generated regions will be incorrect

Reply:
Same as https://github.com/googlefonts/fontc/issues/407

Reply:
Noted on #407. Baby steps :)